concepticondata/conceptlists.tsv

```diff
@@ -388,4 +388,5 @@ Goz-2017-600	Göz, İlyas and Tekcan, Ali İ. and Erciyes, Aslı Aktan	2017		600
 Feleke-2021-150	Feleke, Tekabe Legesse	2021		150	specific, areal	English	Ethiosemitic languages	https://www.sciencedirect.com/science/article/pii/S2215039021000023	Feleke2021		The list was used in a study for the areal classification of Ethiosemitic languages and to establish distances/relationships between the members of this group. According to the author, the distances were calculated by comparing cognates from randomly selected 147 words. Note that the article mentions that the list contains 147 items, whereas the actual supplementary materials consist of 150 items.		
 Klamer-2018-607	Klamer, Marian and Kaiping, Gereon	2018		607	basic	English	Timor-Alor-Pantar languages	https://lexirumah.model-ling.eu/	Kaiping2018		The LexiRuhma database offers translations of a basic list of 607 concepts into several Timor-Alor-Pantar languages. The original database already provides links to the Concepticon, but we provide a mapping considering more recent developments here.		LexiRuhma
 Tjuka-2019-28	Tjuka, Annika	2019		28	body parts	English,German,Wolof,Czech,Marathi,Vietnamese,Greek,Persian,Chinese,Khoekhoe,Hungarian,Japanese,Hebrew,Turkish,Malay	English,German,Wolof,Czech,Marathi,Vietnamese,Greek,Persian,Chinese,Khoekhoe,Hungarian,Japanese,Hebrew,Turkish,Malay		Tjuka2019		The list includes 28 body parts. The body parts were elicited across 13 languages (English and German were added for eliciation) for a typological study on body part extensions to objects and landscapes. The study consisted of a line drawing of a human body and 53 pictures illustrating object and landscape parts which were based on 93 body part metaphors.		
+Kornai-2015-1400	Kornai, András and Ács, Judit and Márton, Makrai and Nemeskey, Dávid Márk and Pajkossy, Katalin and Recski, Gábor	2015		1400	ratings, norms	English	Latin, Hungarian, Polish	https://hlt.bme.hu/en/projects/4lang	Kornai2015		…		
```

Comment:
Can I ask you to draft a description for the resource, based on the paper study?

Reply:
Yes, might take me a second though. I've yet to read the paper completely and I've yet to understand fully what is happening here, to be honest.